com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -1229,6 +1230,16 @@ void ConfigureKeywords(bool enableBakeShadowMask, HDCamera hdCamera, CommandBuff
 
             // Raise or remove the depth msaa flag based on the frame setting
             CoreUtils.SetKeyword(cmd, "WRITE_MSAA_DEPTH", hdCamera.frameSettings.IsEnabled(FrameSettingsField.MSAA));
+
+#if UNITY_EDITOR && ENABLE_VIRTUALTEXTURES
+            //Set VTDebug mode based on useTexArray
+            if (!m_VTDebugBlit) m_VTDebugBlit = new Material(Shader.Find("Hidden/DebugVTBlit"));
```

Comment:
Hey, 
2 remark here.
Is there any reasons to keep this debug mode editor only? All our debug mode are available at runtime and it is useful for users to see it on device (like no PS4 screen for exampl)e.

Second, we don't inspect shader directly Shader.Find("Hidden/DebugVTBlit")); in HDRP we always put system shader in an hdrp resource asset \com.unity.render-pipelines.high-definition\Runtime\RenderPipeline\RenderPipelineResources.cs  and then access it like this.
This is to not pollute other render pipeline with our thing (like URP).